CONTRIBUTING.md

```diff
@@ -0,0 +1,347 @@
+
+In this article, we will see how to import Swagger API into Postman and use mock server.
+
+###  What is a Postman?
+
+Postman is an API platform for building and using APIs. Postman simplifies each step of the API lifecycle and streamlines collaboration so you can create better APIs—faster.
+
+###  Postman mock server steps
+
+####  Download Postman
+
+Download the [app](https://www.postman.com/downloads) to get started with the Postman API platform.
+
+#### Import API
+
+If you need to use a mock server, please import the [swagger.yaml](https://raw.githubusercontent.com/dragonflyoss/Dragonfly2/main/api/manager/swagger.yaml) file of Dragonfly Manager, refer to [importing-and-exporting-data](https://learning.postman.com/docs/getting-started/importing-and-exporting-data/).
+
+**Step 1:** Import API.
+
+![](/docs/images/contributing/postman-import.png)
+
+**Step 2:** Copy the swagger.yaml URL.
+
+![](/docs/images/contributing/postman-import-api.png)
+
+**Step 3:** Select OpenAPI 2.0 with Postman collection.
+
+![](/docs/images/contributing/postman-select.png)
+
+#### Setting up mock server
+
+To use mock server, please do the following, refer to  [mocking-data-setting-up-mock](https://learning.postman.com/docs/designing-and-developing-your-api/mocking-data/setting-up-mock/).
+
+**Step 1:** Choose **Collections** in the sidebar, choose the more actions icon next to the collection you want to mock, then choose **Mock collection**.
+
+![](/docs/images/contributing/postman-mock-collection.png)
+
+**Step 2:** Enter the name of the mock server and click **Create Mock Server** button.
+
+![](/docs/images/contributing/postman-creat-mock.png)
+
+**Step 3:** Copy the mock server URL.
+
+![](/docs/images/contributing/postman-copy-mockUrl.png)
+
+**Step 4:** Add the mock server URL and click the **Save** button.
+
+ ![](/docs/images/contributing/postman-change-path.png)
+
+**Step 5:** Click the **Send** button to initiate a request.
+
+![](docs/images/contributing/postman-request.png)
+
+**Step 6:** You can verify the success of requests by checking the request logs in the **Mock Servers**.
+
+![](docs/images/contributing/postman-verify.png)
+
+**Step 7:** Export mock server URL through environment variables in project.
+
+Install dependencies.
+
+```shell
+npm install
+```
+
+First you need `export NEXT_PUBLIC_DRAGONFLY_PUBLIC_API=your-mock-server-URL`, where your-mock-server-URL is your own mock server URL, start your server after the import is successful.
+
+```shell
+$ export NEXT_PUBLIC_DRAGONFLY_PUBLIC_API=<your-mock-server-URL>
+$ npm run dev
+> console@1.0.0 dev
+> cross-env NODE_ENV='development' next dev -p 3000
+ready - started server on 0.0.0.0:3000, url: http://localhost:3000
+event - compiled client and server successfully in 2.8s (173 modules)
+```
+
+**Step 8:** If you want to view the successful response case, you need to perform the following operations, modify the name under SignUp user such as **OK**, add parameters, and click Sava.
```

Comment:
If you want **Mock Server** to return **OK response**, you need to add query params with code, so that the request matches the query params, then **OK response** will be returned.